docs/README.md

```diff
@@ -0,0 +1,137 @@
+# letsencrypt
+
+This role issues Let's Encrypt certificates via DNS-01 or HTTP-01 challenge.
```

Comment:
this feels a bit redundant with the root README and we should be clear about using `role` or `collection`

Reply:
I see why you feel like this.
The reason behind this is that a collection can provide multiple roles. As the role is the thing which does all the work and is the only role in this collection i mostly speak in this term in the documentation. Does that sound logical or should we discuss this more in depth?

Reply:
For now that may be the best approach. 

I could imagine, that we include some other roles for rollout of certificates to different targets in the future. But that is in the future. In this context I was thinking about renaming this `README.md` to `role-letsencrypt.md` (or something like this) and use it directly to explain the role. All general Information should then go to the root `README.md` and we are clear on only including stuff specific to this letsencrypt role here. Hopefully with little text duplication. ;)

Reply:
Under this point of view i agree with you that we maybe should be more clear about the wording. I also see the rollout of certificates as one objective. 

As for the current situation i cannot quite wrap my head around a change that would satisfy this right off the bet.